src/controller/python/chip/yaml/format_converter.py

```diff
@@ -23,8 +23,49 @@
 import binascii
 
 
+def _substitute_in_config_variables(field_value, config_values: dict):
+    ''' Substitutes values that are config variables.
+
+    YAML values can contain a string of a configuration variable name. In these instances we
+    substitute the configuration variable name with the actual value.
```

Comment:
Suggest showing example input and equivalent reworked output (i.e. what the equivalent YAML would have been if you had saved back the substituted version of the field

Reply:
Added unittest and comment to docstring indicating examples can be found in unittest